mate-settings-daemon/main.c

```diff
@@ -566,7 +404,7 @@ main (int argc, char *argv[])
 
 #ifdef HAVE_LIBNOTIFY
         if (notify_is_initted ())
-            notify_uninit ();
+                notify_uninit ();
```

Comment:
If you do not make any changes to the code, please do not format it, as you will increase the number of lines to review.